modules/rostests/apitests/win32nt/ntuser/NtUserCreateAcceleratorTable.c

```diff
@@ -0,0 +1,188 @@
+/*
+ * PROJECT:     ReactOS api tests
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Test for NtUserCreateAcceleratorTable
+ * COPYRIGHT:   Copyright 2025 Max Korostil (mrmks04@yandex.ru)
+ */
+
+#include "../win32nt.h"
+#include <pseh/pseh2.h>
+
+#define MIN_VALID_NUMBER 1
+#define MAX_VALID_NUMBER 32767
+
+START_TEST(NtUserCreateAcceleratorTable)
+{
+    HACCEL hAccel = NULL;
+    ACCEL Entries[5] = {0};
+    ULONG EntriesCount = 0x80000005;
+    BOOL bHung = FALSE;
+    LPACCEL pEntries = NULL;
+
+    /* Try heap overflow */
+    SetLastError(0xdeadbeef);
+    _SEH2_TRY
+    {
+        hAccel = NtUserCreateAcceleratorTable(Entries, EntriesCount);
+    }
+    _SEH2_EXCEPT(EXCEPTION_EXECUTE_HANDLER)
+    {
+        bHung = TRUE;
+    }
+    _SEH2_END;
+
+    ok_long(GetLastError(), 0xdeadbeef);
+    ok_int(bHung, FALSE);
+    ok_hdl(hAccel, NULL);
+
+    /* Try NULL Entries argument */
+    SetLastError(0xdeadbeef);
+    bHung = FALSE;
+    _SEH2_TRY
+    {
+        hAccel = NtUserCreateAcceleratorTable(NULL, MIN_VALID_NUMBER);
+    }
+    _SEH2_EXCEPT(EXCEPTION_EXECUTE_HANDLER)
+    {
+        bHung = TRUE;
+    }
+    _SEH2_END;
+
+    ok_long(GetLastError(), 0xdeadbeef);
+    ok_int(bHung, FALSE);
+    ok_hdl(hAccel, NULL);
+
+    /* Try invalid Entries argument */
+    SetLastError(0xdeadbeef);
+    bHung = FALSE;
+    _SEH2_TRY
+    {
+        hAccel = NtUserCreateAcceleratorTable((LPACCEL)(ULONG_PTR)0xC0FEC0FE, MIN_VALID_NUMBER);
+    }
+    _SEH2_EXCEPT(EXCEPTION_EXECUTE_HANDLER)
+    {
+        bHung = TRUE;
+    }
+    _SEH2_END;
+
+    ok_long(GetLastError(), ERROR_NOACCESS);
```

Comment:
Interesting that we get an actual `LastError` here: I would guess this is a side-effect.
Before removing it/them, let's see what `EngGetLastError()` returns.